src/compas/geometry/_core/tangent.py

```diff
@@ -26,16 +26,17 @@ def tangent_points_to_circle_xy(circle, point):
     Examples
     --------
     >>> from compas.geometry import allclose
-    >>> circle = (0, 0, 0), 1.0
+    >>> circle = ((0, 0, 0), (1, 0, 0), (0, 1, 0)), 1.0
```

Comment:
this line is causing the issue...
should be
```python
>>> circle = ((0, 0, 0), (0, 0, 1)), 1.0
```

Reply:
I have updated the dogstring to the correct circle definition